Packages/cdms2/Lib/grid.py

```diff
@@ -496,14 +496,6 @@ def genBounds(self):
         else:
             latbnds = lat.genGenericBounds()
 
-        # Stretch latitude bounds to +/- 90.0
-        if ascending:
-            latbnds[0,0] = min(latbnds[0,0],-90.0)
-            latbnds[-1,1] = max(latbnds[-1,1],90.0)
-        else:
-            latbnds[0,0] = max(latbnds[0,0],+90.0)
-            latbnds[-1,1] = min(latbnds[-1,1],-90.0)
-
         # Get longitude bounds
```

Comment:
With the new _genericBounds_flag, I got a different average for a few tests.
The reason was that bounds were generated in grid.genBounds, and modified in the offending code which I removed, instead of axis.genGenericBounds. Also see the following changed baseline (first image).
https://github.com/UV-CDAT/uvcdat-testdata/pull/126/files
Image 4 does not have the purple lines which I think should not be there - all those images are different kind of interpolations.